.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged && github.base_ref == 'master'
```

Comment:
also this ```   if: "! contains(github.event.head_commit.message, '[skip ci]')"``` which i got from @JAdshead PR https://github.com/americanexpress/fetch-enhancers/pull/6